README.md

```diff
@@ -1 +1,60 @@
-# kex
\ No newline at end of file
+The purpose of kex is to provide an opinionated executor into a kubernetes container.  Traditionally, if one wanted to exec into a kubernetes container, one had to `kubectl get pods --namespace foo`, visually identify the pod of interest, copy that pod to the buffer, and then `kubectl --namespace foo exec -it <paste_buffer> bash` to exec in to the pod.  This simple utility aims to provide a namespace-specific pod selector for quick execution.
+
+# kex
+
+kex is a pod exec utility.
+
+```
+USAGE:
+  kex             : exec in to the default pod
+  kex ls,list     : list the available pods
+  kex <NUM>       : exec into a specified pod
```

Comment:
This operates on the same assumption as K8S, that if you have multiple containers in a pod, it defaults to the one that is not a sidecar.  Falls over if two primary containers exist in a pod (but that's bad practice, and should be discouraged from a microservices perspective).  For clarity, these are pods listed, not containers in a pod.